force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,47 +400,50 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
-
-        // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
+        Map <String, Integer> empSalary = new Map<String,Integer>{'John Doe'     => 50000,
+                                                                  'Jane Smith'   => 60000,
+                                                                  'Sam Brown'    => 55000,
+                                                                  'Alice Johnson'=> 65000
+        };// See above key-value pairs in the Map    
 
         // Get the salary of the employee
-
         // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
-    }
+        return empSalary.get(employeeName);
+}
 
-    /**
-     * Question 20
-     * Create a map where the keys are the names of employees and the values are their salaries.
-     * Then, iterates over the map and checks if an employee's salary is more than 55000.
-     * If it is, add the employee's name to a list of employees that is returned by the method.
-     * Remember that a Map is a collection of key-value pairs. Key are a set of unique values, so you can't have duplicate keys. Values are a list of values, so you can have duplicate values.
-     * Resource: https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_loops_for_lists.htm
-     * Example: getHighPaidEmployees() should return ['Jane Smith', 'Alice Johnson']
-     * @return A list of employee names who have a salary more than 55000.
-     */
-    public static List<String> getHighPaidEmployees() {
-        // Initialize a Map
-        Map<String, Integer> employeeSalaries = new Map<String, Integer>();
+/**
+ * Question 20
+ * Create a map where the keys are the names of employees and the values are their salaries.
+ * Then, iterates over the map and checks if an employee's salary is more than 55000.
+ * If it is, add the employee's name to a list of employees that is returned by the method.
+ * Remember that a Map is a collection of key-value pairs. Key are a set of unique values, so you can't have duplicate keys. Values are a list of values, so you can have duplicate values.
+ * Resource: https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_loops_for_lists.htm
+ * Example: getHighPaidEmployees() should return ['Jane Smith', 'Alice Johnson']
+ * @return A list of employee names who have a salary more than 55000.
+ */
+public static List<String> getHighPaidEmployees() {
+// Initialize a Map
+Map<String, Integer> employeeSalariesMap = new Map<String, Integer>{'John Doe'  => 50000,
```

Comment:
Since employeeSalariesMap and empSalary have the same values, maybe you can make it a static variable and call it for both methods that way you don't have to write the same map twice.